lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -8,45 +8,35 @@
 import logging
 import os
 
+import h5py
+import numpy as np
 import requests
+import yaml
 from gxformat2._yaml import ordered_dump
+from keras.models import model_from_json
 from markupsafe import escape
 from sqlalchemy import desc, false, or_, true
 from sqlalchemy.orm import joinedload
 
-from galaxy import (
-    exceptions,
-    model,
-    util
-)
-from galaxy.managers import (
-    histories,
-    workflows
-)
+from galaxy import exceptions, model, util
+from galaxy.managers import histories, workflows
```

Comment:
We prefer to have the `from ...` imports on separate lines to reduce merge conflicts. Can you revert these changes?

Reply:
Fixed them. Thanks a lot!